### PR TITLE
Homework4: Alexander Zhuk

### PR DESCRIPTION
**Pull request by ZhukAlexander90**

### CODE REVIEW

File: Homework4_ZhukAlexander/task2_cities.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"""
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task2_cities.py:1]: what about Brest in Belarus and France?

File: Homework4_ZhukAlexander/task3_num_list1.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+first_list = []
+second_list = []
+while len(first_list) < 20:
+    new_num = random.randrange(20)
+    first_list.append(new_num)
+    new_num = random.randrange(20)
+    second_list.append(new_num)
+print("первый список чисел: ", first_list)
+print("второй список чисел: ", second_list)
+# print(set(first_list))
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task3_num_list1.py:18]: do we need these comments?

File: Homework4_ZhukAlexander/task4_num_list2.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+first_list = []
+second_list = []
+while len(first_list) < 20:
+    new_num = random.randrange(20)
+    first_list.append(new_num)
+    new_num = random.randrange(20)
+    second_list.append(new_num)
+print("первый список чисел: ", first_list)
+print("второй список чисел: ", second_list)
+# print(set(first_list))
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task4_num_list2.py:18]: do we need these comments?

File: Homework4_ZhukAlexander/task5_languages.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Italian
+French
+
+"""
+
+student_list = [{input("Язык школьника: ") for qnt_languages in
+                range(int(input("Количество языков: ")))}
+                for qnt_students in range(int(input("Количество "
+                                                    "школьников: ")))]
+# print(student_list)
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task5_languages.py:30]: do we need it?

File: Homework4_ZhukAlexander/task6_ words.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+Во входной строке записан текст. Словом считается последовательность
+непробельных символов идущих подряд, слова разделены одним или большим
+числом пробелов или символами конца строки. Определите, сколько различных
+слов содержится в этом тексте.
+
+"""
+
+import string
+new_str = input("Текст (может быть со знаками): ")
+func = str.maketrans(dict.fromkeys(string.punctuation))
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task6_ words.py:11]: we don't need puctuation according to task description ) but overall it's cool

File: Homework4_ZhukAlexander/task7_evklid.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+print(first_num)
+
+first_num = int(input("Первое число: "))
+second_num = int(input("Второе число: "))
+while first_num != 0 and second_num != 0:       # нахождение НОД делением
+    if first_num > second_num:
+        first_num %= second_num
+    else:
+        second_num %= first_num
+print(first_num + second_num)
```
Comment (belovmd) [inline Homework4_ZhukAlexander/task7_evklid.py:23]: it's a cheat. we don't need sum according to algorithm.

**Review by belovmd — CHANGES_REQUESTED** [review]
Please address next issues:

File: Homework4_ZhukAlexander/task4_num_list2.py (hunk 3 above)
Comment (ZhukAlexander90) [reply Homework4_ZhukAlexander/task4_num_list2.py:18]: No, I left it to myself

File: Homework4_ZhukAlexander/task3_num_list1.py (hunk 2 above)
Comment (ZhukAlexander90) [reply Homework4_ZhukAlexander/task3_num_list1.py:18]: I left it to myself. for check

File: Homework4_ZhukAlexander/task5_languages.py (hunk 4 above)
Comment (ZhukAlexander90) [reply Homework4_ZhukAlexander/task5_languages.py:30]: not necessary
need to delete?

**Review by belovmd — COMMENTED** [review]
7
overall it's inaccurate with redundant comments + task with cities has issues

**Pull request closed without merging**